database/database.go

```diff
@@ -7,6 +7,36 @@ import (
 	"github.com/wtg/shuttletracker/model"
 )
 
+//RouteIsActive determines if a given route is active based on its schedule intervals
+func RouteIsActive(r *model.Route) {
```

Comment:
Can we rename this? Maybe `SetRouteActivationStatus` or something to indicate that it modifies the route instead of returning something.